application/views/registration_individual.php

```diff
@@ -1,105 +1,149 @@
+        </div>
+        <div class="show-if-attend">
+            <div class="row">
+                <label class="col-sm-1">个人赛</label>
+                <div class="col-sm-2">
+                    <select class="form-control" name="race">
+                        <?php foreach ($GLOBALS['CAPURACE'] as $key => $value): ?>
+                            <option value="<?=$key?>"><?=$value?></option>
+                        <?php endforeach; ?>
+                    </select>
+                </div>
+                <label class="col-sm-1">团体赛</label>
+                <div class="col-sm-1">
+                    <input type="checkbox" name="ifteam">
+                </div>
+                <div class="show-if-race">
+                    <label class="col-sm-1">赠送项目</label>
+                    <label class="col-sm-1">5.16公路</label>
+                    <div class="col-sm-2">
+                        <select class="form-control" name="shimano16">
+                            <?php foreach ($GLOBALS['SHIMANO_RDB'] as $key => $value): ?>
+                                <option value="<?=$key?>"><?=$value?></option>
+                            <?php endforeach; ?>
+                        </select>
+                    </div>
+                    <label class="col-sm-1">5.17山地</label>
+                    <div class="col-sm-2">
+                        <select class="form-control" name="shimano17">
+                            <?php foreach ($GLOBALS['SHIMANO_MTB'] as $key => $value): ?>
+                                <option value="<?=$key?>"><?=$value?></option>
+                            <?php endforeach; ?>
+                        </select>
+                    </div>
+                </div>
             </div>
-            <br/>
-            <br/>
         </div>
-        <?php endforeach; ?>
     </div>
     <hr/>
-    <div class="col-sm-3">
-        <button class="btn btn-info btn-block" id="return-to-index">返回帮助页面</button>
-    </div>
-    <div class="col-sm-3">
-        <button class="btn btn-primary btn-block" onclick="addIndividual()">添加一个队员</button>
+    <div class="col-sm-4"></div>
+    <div class="col-sm-4">
+        <button class="btn btn-warning btn-block" id="cache">保存</button>
     </div>
+    <div class="col-sm-4"></div>
+</div>
+<!-- The code above creates the individual form -->
+
+<div class="indcontainer ind-list">
+    <p><h3>已保存人员列表</h3></p>
+    <hr/>
+    <table class="table table-hover">
+        <thead>
+        <tr>
+            <th>序号</th>
+            <th>姓名</th>
+            <th>性别</th>
+            <th>手机号</th>
+            <th>身份证号</th>
+            <th>北大赛</th>
+            <th>公路日</th>
+            <th>山地日</th>
+            <th>住宿</th>
+            <th>5.16晚餐</th>
+            <th>5.17午餐</th>
+            <th>清真</th>
+            <th>操作</th>
+        </tr>
+        </thead>
+        <tbody>
+        <!-- Here should be some PHP foreach statements to display the athletes.
```

Comment:
I prefer we still have a hidden pattern to add the result, but it's OK now. You can do this in following PR.